### PR TITLE
[environment-variables.mdx] link to `astro:env`

### PR DESCRIPTION
**Pull request by sarah11918**
#### Description (required)

To help with search results and discoverability for `astro:env`, adds a link from the main Environment Variables guide.

### CODE REVIEW

File: src/content/docs/en/guides/environment-variables.mdx [hunk 1]
```diff
@@ -5,6 +5,10 @@ i18nReady: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
+:::tip[Experimental `astro:env`]
+Find out more about the [experimental `astro:env` API](/en/reference/configuration-reference/#experimentalenv) for type-safe environment variables!
+:::
```
Comment (sarah11918) [inline src/content/docs/en/guides/environment-variables.mdx:10]:
```suggestion
:::tip[Looking for astro:env?]
Find out more about the [experimental `astro:env` API](/en/reference/configuration-reference/#experimentalenv) for type-safe environment variables!
:::
```

**Review by Fryuni — APPROVED** [review]

File: src/content/docs/en/guides/environment-variables.mdx [hunk 2]
```diff
@@ -5,6 +5,10 @@ i18nReady: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
+:::tip[Looking for astro:env?]
```
Comment (TheOtterlord) [inline src/content/docs/en/guides/environment-variables.mdx:8]:
```suggestion
:::tip[Looking for astro&#58;env\?]
```

**Review by Adammatthiesen — APPROVED** [review]
ASTRO ENV TAKEOVER!

**Review by BryceRussell — APPROVED** [review]
LGTM!

**Review by TheOtterlord — APPROVED** [review]

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**